Sloth.Web/Controllers/TransactionsController.cs

```diff
@@ -19,13 +20,38 @@ public TransactionsController(ApplicationUserManager userManager, SlothDbContext
         // GET: /<controller>/
         public async Task<IActionResult> Index()
         {
+            return await Filtered(DateTime.Now.AddMonths(-1), DateTime.Now);
+        }
+
+        public async Task<IActionResult> Filtered(DateTime @from, DateTime to)
+        {
+            var fromUtc = DateTime.SpecifyKind(@from, DateTimeKind.Local).ToUniversalTime().Date;
```

Comment:
Have not used SpecifyKind before, will need to make sure it works in the deployed version.

Reply:
Now that I think about it, it's not needed, since `.ToUniversalTime()` assumes an unspecified is local and `.ToLocalTime()` assumes an unspecified is universal.